rancher/v1.6/en/installing-rancher/installing-server/no-internet-access/index.md

```diff
@@ -137,3 +137,21 @@ After accessing the UI, you can click on the **Add Host** button. This will imme
 The cloud providers will not work as Rancher uses `docker-machine` to provision the hosts through the cloud providers. Click on the **Custom** icon to add the host.
 
 The command from the UI can be used on any machine that has Docker configured to use HTTP proxy.
+
+### Catalogs
+On every release, the `rancher/server` container will be built containing a cached copy of the [Rancher catalog](https://github.com/rancher/rancher-catalog) and [community catalog](https://github.com/rancher/community-catalog) belonging to that release.
+
+#### Rancher catalog
+As we only support one release of [infrastructure services]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/rancher-services/) per Rancher release, you do not have to mirror or update the Rancher catalog to be supported or up-to-date.
```

Comment:
This is not completely true, as we do sometimes add in infrastructure releases that are  not tied to a specific Rancher release. We should always recommend to mirror the catalog to make sure you get the latest.

Reply:
Added